cli/util/glob.rs

```diff
@@ -248,9 +248,11 @@ impl GlobPattern {
   }
 
   pub fn new(pattern: &str) -> Result<Self, AnyError> {
-    let pattern =
-      glob::Pattern::new(&escape_brackets(pattern).replace('\\', "/"))
-        .with_context(|| format!("Failed to expand glob: \"{}\"", pattern))?;
+    let pattern = escape_brackets(pattern)
+      .replace('\\', "/")
+      .replace("/./", "/");
```

Comment:
Wouldn't it be better to construct each time the absolute path from the relative path instead of building a relative path compliant pattern? 
I do not know what is the exact expected usage of that glob expansion code but that does not look right to do path munging in code dedicated to glob handling.

Reply:
Yeah that's what's done in the real fix in https://github.com/denoland/deno/pull/21925

It requires a much larger refactor to do that though.